### PR TITLE
Split PanelContainer from SpeciesDetailsPanel

### PR DESCRIPTION
**Pull request by 84634E1A607A**
## This PR wants to be merged into [more_fossilisation_buttons](https://github.com/Revolutionary-Games/Thrive/tree/more_fossilisation_buttons) (#3949)

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [ ] Initial code review passed (this and further items should not be checked by the PR author)
- [ ] Functionality is confirmed working by another person (see above checklist link)
- [ ] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

**Comment by 84634E1A607A** [comment]
I think the species details part can still be extracted to its own scene. Now FossiliseDialog still has duplicates to DetailsPanel.

**Comment by hhyyrylainen** [comment]
I'll wait until this is merged or rejected until I review https://github.com/Revolutionary-Games/Thrive/pull/3963

File: project.godot
Comment (Oliveriver) [inline project.godot:447]: Is there a reason this change is here?

**Review by Oliveriver — APPROVED** [review]
One small thing but seems fine and works otherwise.

File: project.godot
Comment (84634E1A607A) [reply project.godot:447]: My Godot always wants to add it. And hh seems to have accepted it and merged these lines into master in my autoevo II pr.

**Pull request merged**
